src/common/tests_dmg_helpers.c

```diff
@@ -1393,7 +1393,7 @@ dmg_storage_set_nvme_fault(const char *dmg_config_file,
 			D_GOTO(out, rc = -DER_NOMEM);
 	}
 
-	args = cmd_push_arg(args, &argcount, " --host-list=%s ", host);
```

Comment:
This function is only used by `src/tests/suite/daos_nvme_recovery.c`
Which is ran by `DaosCoreTestNvme`

Since only documentation and that one test are affected, I think:
```
Test-tag: DaosCoreTestNvme
```